enclave-tls/src/tls_wrappers/openssl/init.c

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2020-2021 Alibaba Cloud and Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <enclave-tls/log.h>
+#include <enclave-tls/tls_wrapper.h>
+#include "openssl.h"
+
+tls_wrapper_err_t openssl_tls_init(tls_wrapper_ctx_t *ctx)
+{
+	ETLS_DEBUG("ctx %p\n", ctx);
+
+	if (!ctx)
+		return -TLS_WRAPPER_ERR_INVALID;
+
+	OpenSSL_add_all_algorithms();
```

Comment:
It is not necessary to load all algo. Check the manual for the necessary algo for TLS only.